wbstack/src/Settings/LocalSettings.php

```diff
@@ -525,13 +525,35 @@ function onPersonalUrlsConfirmAccount( array &$personal_urls, Title $title, Skin
     }
 }
 
-# CirrusSearch etc.
-// wfLoadExtension( 'Elastica' );
-// wfLoadExtension( 'CirrusSearch' );
-// wfLoadExtension( 'WikibaseCirrusSearch' );
-// if ( $wikiInfo->getSetting('wwExtEnableWikibaseLexeme') ) {
-//     wfLoadExtension('WikibaseLexemeCirrusSearch');
-// }
+// ElasticSearch extension loading
+// Allow maintainance scripts to enter this for the localization cache to be built
+if ( $wikiInfo->getSetting( 'wwExtEnableElasticSearch' ) || $wwDomainIsMaintenance ) {
```

Comment:
the wwDomainIsMaintenance bit is probably not needed now because of https://github.com/wbstack/mediawiki/pull/106/commits/e1f17b5e417968428e65893d27b98aeaa936479f